src/content/architecture/_index.en.md

```diff
@@ -1,7 +1,21 @@
----
-title: "Architecture"
-date: 2020-02-19T21:00:30+01:00
-pre: "<b>3. </b>"
-weight: 15
----
++++
+title = "Architecture"
```

Comment:
@tpantelis in the commit message of this PR, can you include
Fixes issue# https://github.com/submariner-io/submariner-website/issues/18

Reply:
Done